### PR TITLE
Auto build and upload wheel for nightly + release

### PR DESCRIPTION
**Pull request by KSGulin**
Adds ~two~ one GitHub Actions workflow

**Upload Nightly Wheel**
Trigger: Push onto master
Action: Build a new pip wheel and upload it to the [nightly release](https://github.com/neuralmagic/yolov5/releases/tag/nightly)

_Release based automation deferred for now_
~**Upload Release Wheel**
Trigger: Push to a release tag (any tag matching `v*`)
Action: Create a new release. Build a new pip wheel and upload it to the new release.~

### CODE REVIEW

**Review by rahul-tuli — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -8,8 +8,6 @@ Pillow>=7.1.2
 PyYAML>=5.3.1
 requests>=2.23.0
 scipy>=1.4.1
-torch>=1.7.0
```
Comment (rahul-tuli) [inline requirements.txt:11]: Is this intentional?

**Review by rahul-tuli — APPROVED** [review]

**Review by bfineran — APPROVED** [review]
great PR, let's monitor function closely after landing

File: requirements.txt (hunk 1 above)
Comment (KSGulin) [reply requirements.txt:11]: Yep. The torch requirements are already filled by sparseml further down in the file, so these get overwritten anyway.

**Pull request merged**